.github/workflows/test-admin-interface.yml

```diff
@@ -0,0 +1,97 @@
+name: Gradido Admin Interface Test CI
+
+on:
+  push:
+    branches:
+      - separate-admin-interface-workflow
+
+jobs:
+  # only (but most important) job from this workflow required for pull requests
+  # check results serve as run conditions for all other jobs here 
+  files-changed:
+    name: Detect File Changes - Admin Interface
```

Comment:
```suggestion
    name: Detect File Changes - ${service}
```

think about it or just close

Reply:
![image](https://user-images.githubusercontent.com/1238238/221151537-ba9a5771-e028-472d-9648-41bc4ef9994f.png)

https://docs.github.com/en/actions/learn-github-actions/variables

Reply:
We'll skip this for now.
Accessing env variables in a workflow does not work as described (see [this discussion](https://github.com/orgs/community/discussions/25295))